boards/BMS/Inc/Io/Io_CurrentSense.h

```diff
@@ -11,7 +11,7 @@
  * @return EXIT_CODE_OUT_OF_RANGE if adc_voltage is negative
  *         EXIT_CODE_INVALID_ARGS if low_res_main_current is NULL
```

Comment:
i'd also update these since we're not returning `ExitCode` anymore

Reply:
updated